assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/MetricsOverrideBinding.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.deploy;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import java.util.concurrent.ExecutorService;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import org.eclipse.che.api.deploy.jsonrpc.CheMajorWebSocketEndpointConfiguration;
+import org.eclipse.che.api.deploy.jsonrpc.CheMajorWebSocketEndpointExecutorServiceProvider;
+import org.eclipse.che.api.deploy.jsonrpc.CheMinorWebSocketEndpointConfiguration;
+import org.eclipse.che.api.deploy.jsonrpc.CheMinorWebSocketEndpointExecutorServiceProvider;
+import org.eclipse.che.core.metrics.ExecutorServiceMetrics;
+
+/**
```

Comment:
Don't know what is right way to send multiliness suggestion.
Anyway
```suggestion
/**
 * {@link Module} that provide alternative implementation of {@link
 * CheMajorWebSocketEndpointExecutorServiceProvider} and {@link
 * CheMinorWebSocketEndpointExecutorServiceProvider}.
 *
 * <p>{@link ExecutorService} that is returned by these providers will publish metrics to {@link
 * PrometheusMeterRegistry}
 */
```